financial-orc/graphs/tools/database_retriever.py

```diff
@@ -5,6 +5,26 @@
 import json
 from collections import OrderedDict
 from langchain_core.documents import Document
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.secrets import SecretClient
+import logging
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
@namtran6701 is this required wouldn't work with out it? just setting up the environment var in the local.setting file